src/util.rs

```diff
@@ -45,13 +45,17 @@ pub const NO_LIMIT: u64 = u64::MAX;
 /// ];
 ///
 /// assert_eq!(entries.len(), 5);
-/// limit_size(&mut entries, 220);
+/// limit_size(&mut entries, Some(220));
 /// assert_eq!(entries.len(), 2);
 /// ```
```

Comment:
It would be good to document the edge case that entries will always have size at least one or use an assertion here to prevent max size being 0.